cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -35,6 +39,10 @@
 
   public static final String LOGICAL_START_TIME = "logical.start.time";
 
+  public static final String START_TIME = "start.time";
+
+  public static final String END_TIME = "end.time";
```

Comment:
nit: better name the value as `endTime` and `startTime` to match with all of other value formats.